src/main/kotlin/org/taktik/icure/domain/filter/impl/maintenancetask/MaintenanceTaskAfterDateFilter.kt

```diff
@@ -11,9 +11,10 @@ import org.taktik.icure.entities.MaintenanceTask
 @KotlinBuilder
 data class MaintenanceTaskAfterDateFilter(
 	override val desc: String? = null,
+	override val healthcarePartyId: String? = null,
 	override val date: Long
 	): AbstractFilter<MaintenanceTask>, MaintenanceTaskAfterDateFilter {
 	override fun matches(item: MaintenanceTask): Boolean {
-		return ((item.created ?: 0) > date)
+		return (healthcarePartyId == null || item.delegations.keys.contains(healthcarePartyId)) && ((item.created ?: 0) > date)
```

Comment:
```suggestion
		return (healthcarePartyId == null || item.delegations.containsKey(healthcarePartyId)) && ((item.created ?: 0) > date)
```
There's a short-hand for that in Kotlin (Forgot to mention it yesterday for the CSS 😓 )
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-map/contains-key.html